help-docs/marketplace/apps/give.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 13
+title: Give by SKALE
+id: give
+sidebar_label: Give
+---
+
+import img1 from '@site/static/img/tutorial/marketplace/poap.png';
+
+## Overview
+
+Send and receive tokens on the SKALE blockchain.
+
+### About this app
+
+Give allows community members to give and receive tokens onchain on the SKALE blockchain. Tokens are sent between members' [gmPASS-controlled smart accounts](/help-docs/key-features/gm-pass), creating a personal web of onchain connections between community members.
+
+Give replaces the sunset Kchannel `/tip send` commands.
+
+### Getting Started
+
+<!-- 1. Install the Give Miniapp from the [Collab.Land Marketplace](https://cc.collab.land).
+
+2. -->
+
+### Commands available:
+
+There is one command `/give` with four total subcommands.
+
+| Command | Additional Parameters | Description |
+| --- | --- | --- |

```

Comment:
There's some issue in the table rendering:
![image](https://github.com/abridged/collabland-dev/assets/54375111/a103b2f7-6e5d-4b85-b2a6-078f451aef12)

Reply:
what is the issue?

Reply:
what is the issue @gitaalekhyapaul?

Reply:
ohhhhhhhhhhhhhh its actually displaying `---------------` is the issue nm

Reply:
There's the top row in the table with a bunch of `-`